jumpscale/packages/tfgrid_solutions/scripts/threebot/minimal_entrypoint.py

```diff
@@ -157,3 +164,16 @@
 
 j.sals.fs.mkdirs("/root/.kube")
 j.sals.fs.write_file("/root/.kube/config", KUBE_CONFIG)
+
+# Register provisioning and prepaid wallets
+
+wallet = j.clients.stellar.new(
```

Comment:
we can use `.get()` in case we ran the entryponint more than once, 
alse wallets needs activation. if it's not activated it'll be just an object

Reply:
we have changed new to get but these wallets have been created already with the VDC and we just use the secret to get them so they should be activated already.